src/css/states/states.less

```diff
@@ -172,6 +179,34 @@ body .jwplayer.jw-state-error {
   }
 }
 
+.jw-state-idle,
+.jwplayer.jw-state-complete:not(.jw-flag-casting),
+body .jwplayer.jw-state-error {
+  &.jw-flag-audio-player {
```

Comment:
There is only one state (error) that needs fixing. idle and complete are fine and should not be given new 2. 

> Poster shwon in controlbar-only mode
`body .jwplayer.jw-state-error .jw-preview { display: block; }`